Dockerfile.bmv2

```diff
@@ -16,14 +16,14 @@ ARG MAKEFLAGS=-j2
 ARG IMAGE_TYPE=build
 
 # Select the compiler to use.
-# We install the default version of GCC (GCC 5), as well as clang 3.8 and clang 8.
+# We install the default version of GCC (GCC 9), as well as clang 8 and clang 10.
 ARG CC=gcc
 ARG CXX=g++
 
 ENV PI_DEPS automake \
             build-essential \
-            clang-3.8 \
             clang-8 \
+            clang-10 \
             clang-format-8 \
```

Comment:
> Can't run 'clang-format', please make sure it is installed 

The [`clang-format-10`](https://packages.ubuntu.com/focal/clang-format-10) package might fix this error